src/api/mockDb.ts

```diff
@@ -5,27 +5,32 @@ let mockDb: Todo[] = [
   {
     id: uuid(),
     name: 'Finish writing and reviewing the report by Tomorrow',
-    createdAt: Date.now(),
+    createdAt: new Date(2025, 0, 15, 14, 0, 0),
```

Comment:
It's easier to read, please also change also the others:
```suggestion
    createdAt: new Date('2025-01-15:14:00'),
```